modules/dfpAdServerVideo.js

```diff
@@ -85,6 +114,32 @@ export default function buildDfpVideoUrl(options) {
   });
 }
 
+/**
+ * Builds a video url from a base dfp video url and a winning bid, appending
+ * Prebid-specific key-values.
+ * @param {Object} components base video adserver url parsed into components object
+ * @param {Object} bid winning bid object to append parameters from
```

Comment:
Thanks to @rmloveland's documentation in `src/prebid.js`, you can call this an `AdapterBidResponse` now.